Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/debuglog/upload/server/auth/LogUploadAuthApiV1.kt

```diff
@@ -0,0 +1,22 @@
+package de.rki.coronawarnapp.bugreporting.debuglog.upload.server.auth
+
+import com.google.gson.annotations.SerializedName
+import de.rki.coronawarnapp.server.protocols.internal.ppdd.ElsOtpRequestAndroid
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface LogUploadAuthApiV1 {
+
+    data class AuthResponse(
+        @SerializedName("expirationDate") val expirationDate: String
+    )
+
+    data class AuthError(
```

Comment:
is this ever used?

Reply:
Currently not, but as the server side is not done yet and this is what it could return according to spec, I'd leave it in for now. Once servers are up and running we'll have to recheck this part anyways.